tests/solids/diamondC_2x1x1_pp/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@
                     "${CMAKE_SOURCE_DIR}/tests/solids/diamondC_2x1x1_pp"
                     qmc_short
                     qmc_short_vmcbatch_4b.in.xml
-                    1 4
```

Comment:
Please do not change this test. It is intended to run 1 MPI with 4 threads 16 walkers in 4 crowds. If you change the number of  MPI, the amount of statistics change and the reference numbers must be updated.

Reply:
Also added a comment so I don't do this again before or after the coffee is in effect.